### PR TITLE
Add scope video

### PR DESCRIPTION
**Pull request by jessiwright**
New video from Olivia
Also adding to asset inventory doc after discussion with Martijn

### CODE REVIEW

File: docs/organizations/asset-inventory.md [hunk 1]
```diff
@@ -105,3 +105,6 @@ Your Attack Surface Coverage dashboard gives an overview of your entire attack s
 Asset Submissions allows hackers to submit potentially missed assets for review. You can then accept or reject these assets as part of your organization. If you accept a hacker’s submission, they will then be invited to submit asset enrichment to provide more details about technology tags, CVSS environmental score, and maximum severity.
 
 ![asset submissions](./images/asset-inventory-11.png)
+
+### Asset Tutorial
+<div style="position: relative; padding-bottom: 61.43344709897611%; height: 0;"><iframe src="https://www.loom.com/embed/d9b5fdf65c4c43e0b8f46bf9cabd39b3?sid=cc550dcf-9ff2-4918-8502-3b94b2363359" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```
Comment (martijnrusschen) [inline docs/organizations/asset-inventory.md:110]: Let's try to get a youtube video in here.

Comment (jessiwright) [reply docs/organizations/asset-inventory.md]: New link added! @martijnrusschen

File: docs/organizations/defining-scope.md [hunk 2]
```diff
@@ -41,4 +41,4 @@ Instruction | If you have any detail descriptions or comments on the asset, thi
   * Do data analysis per domain - Is it time to switch your marketing site provider, because all your vulnerabilities stem from that area? Is one particular engineering team responsible for 90% of your Cross-Site Scripting vulnerabilities?
 
 ### Tutorial Video
-<iframe id="ytplayer" type="text/html" width="640" height="360" src="https://www.youtube-nocookie.com/embed/4gK2_OAmiwI" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/H2ZcSP04Vzw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (danMateer) [reply docs/organizations/defining-scope.md:44]: I don't have this running locally so it's worth gut-checking me on it. It should look like this:
image

[the remark this answers is not in the document]
Comment (martijnrusschen) [reply docs/organizations/defining-scope.md:44]: Yes, please use the old nocookie approach.

**Review by martijnrusschen — CHANGES_REQUESTED** [review]

File: docs/organizations/asset-inventory.md [hunk 3]
```diff
@@ -105,3 +105,6 @@ Your Attack Surface Coverage dashboard gives an overview of your entire attack s
 Asset Submissions allows hackers to submit potentially missed assets for review. You can then accept or reject these assets as part of your organization. If you accept a hacker’s submission, they will then be invited to submit asset enrichment to provide more details about technology tags, CVSS environmental score, and maximum severity.
 
 ![asset submissions](./images/asset-inventory-11.png)
+
+### Asset Tutorial
+<iframe width="560" height="315" src="https://www.youtube.com/embed/H2ZcSP04Vzw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (martijnrusschen) [inline docs/organizations/asset-inventory.md:110]: Can we fix this one too to use the no cookie code?

**Review by martijnrusschen — CHANGES_REQUESTED** [review]

**Review by jessiwright — COMMENTED** [review]
changes made

**Review by martijnrusschen — APPROVED** [review]

**Pull request merged**